firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -90,6 +96,17 @@ final class HomepageViewController: UIViewController,
         applyTheme()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.8) { [weak self] in
```

Comment:
Why the timing?

Reply:
This is just how the logic was on the old homepage. I was hoping to chat with andy about if we still wanted that?

Reply:
Perfect!
Also, why are you answering on a SUNDAY! lolololol